src/test/java/io/confluent/examples/streams/interactivequeries/WordCountInteractiveQueriesExampleTest.java

```diff
@@ -147,12 +140,14 @@ public void shouldDemonstrateInteractiveQueries() throws Exception {
         }
     });
     kafkaStreams.start();
-  
+
+    System.out.println("Streams application service starts at " + baseUrl);
```

Comment:
Do we really want to write to stdout instead of logging?

Reply:
No, it was not intentional.